modules/compatibility/src/test/resources/META-INF/services/org.apache.ignite.plugin.PluginProvider

```diff
@@ -0,0 +1 @@
+org.apache.ignite.compatibility.testframework.plugins.TestCompatibilityPluginProvider
```

Comment:
Is this provider would be available for regular test run?
If yes, we definitely should exclude it from regular test classpath.

Reply:
No, this plugin will be used only in tests in the compatibility-module and won't be used in any other module like ignite-core.